roachtest: increase the verbosity around tests that kills nodes with traffic

iskettaneh opened the pull request:
The PR increases the verbosity around some roachtests that kills nodes while sending traffic. The roachtests affected by this PR:

1. restart/down-for-2m
2. tpcc/multiregion/survive=region/chaos=true
3. tpcc/multiregion/survive=zone/chaos=true
4. tpcc/w=100/nodes=3/chaos=true

References: #138028

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/140490)
<!-- Reviewable:end -->

iskettaneh committed:
roachtest: increase verbosity around tests that kills nodes with traffic

The PR increases the verbosity around some roachtests that kills nodes
while sending traffic.

The roachtests affected by this PR:

1. restart/down-for-2m
2. tpcc/multiregion/survive=region/chaos=true
3. tpcc/multiregion/survive=zone/chaos=true
4. tpcc/w=100/nodes=3/chaos=true

References: #138028

Release note: None

iskettaneh commented:
TFTR!

bors r+

iskettaneh commented:
bors r+

rickystewart commented:
bors r+

rickystewart commented:
bors r+
